CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+# How to contribute to Passwords?
+
+Thank you for your interest in contributing to this Cozy application! There are many ways to contribute, and we appreciate all of them.
+
+
+## Security Issues
+
+If you discover a security issue, please bring it to our attention right away! Please **DO NOT** file a public issue, instead send your report privately to security AT cozycloud DOT cc.
+
+Security reports are greatly appreciated and we will publicly thank you for it. We currently do not offer a paid security bounty program, but are not ruling it out in the future.
+
+
+## Bug Reports
+
+While bugs are unfortunate, they're a reality in software. We can't fix what we don't know about, so please report liberally. If you're not sure if something is a bug or not, feel free to file a bug anyway.
+
+Opening an issue is as easy as following [this link][issues] and filling out the fields. Here are some things you can write about your bug:
+
+- A short summary
+- What did you try, step by step?
+- What did you expect?
+- What did happen instead?
+- What is the version of the Passwords app?
+
+
+## Pull Requests
+
+Please keep in mind that:
+
+- Pull-Requests point to the `master` branch
+- You need to cover your code and feature by tests
+- You may add documentation in the `/docs` directory to explain your choices if needed
+- We recommend to use [task lists][checkbox] to explain steps / features in your Pull-Request description
+- you do _not_ need to build app to submit a PR
+- you should update the Transifex source locale file if you modify it for your feature needs (see [Localization section in README][localization])
+
+
+### Workflow
+
+Pull requests are the primary mechanism we use to change Cozy. GitHub itself has some [great documentation][pr] on using the Pull Request feature. We use the _fork and pull_ model described there.
+
+#### Step 1: Fork
+
+Fork the project on GitHub and [check out your copy locally][forking].
+
+```
+$ git clone github.com/cozycloud/cozy-passwords.git
```

Comment:
It should be github.com/cozy, not github.com/cozycloud, no?

(idem in README.md, package.json, etc.)